.github/workflows/tests.yml

```diff
@@ -0,0 +1,45 @@
+name: Linux tests
+
+on:
+  push:
+    branches:
+      - master
+
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  linux:
+    name: Linux py${{ matrix.PYTHON_VERSION }} tests
+    runs-on: ubuntu-latest
+    env:
+      CI: True
+      PYTHON_VERSION: ${{ matrix.PYTHON_VERSION }}
+    strategy:
+      fail-fast: false
+      matrix:
+        PYTHON_VERSION: ['3.5', '3.6', '3.8']
```

Comment:
Seems like they've already got 3.9: probably better to know sooner! Heck I'd throw at least an pypy in there, too!

Reply:
Yes, will add it!

Reply:
at which point, is 3.6 even going to tell you much more than 3.5 until EOL in september?